### PR TITLE
have R.converge return a function of appropriate length

### PR DESCRIPTION
**Pull request by davidchambers**

### CODE REVIEW

File: src/converge.js [hunk 1]
```diff
@@ -31,10 +34,10 @@ var curryN = require('./curryN');
  */
 module.exports = curryN(3, function(after) {
   var fns = _slice(arguments, 1);
-  return function() {
+  return arity(max(pluck('length', arguments)), function() {
```
Comment (buzzdecafe) [inline src/converge.js]: as with `uncurryN`, should this use `curryN`?

Comment (CrossEye) [inline src/converge.js]: Probably yes for `curryN`, but one other thing:

```javascript
return arity(max(pluck('length', _slice(arguments, 1))), function() {  // or
return curryN(max(pluck('length', _slice(arguments, 1))), function() {
```

The first function -- the join -- should not count when calculating the length of the returned function.

Comment (buzzdecafe) [inline src/converge.js]: simpler:

`return curryN(max(pluck('length', fns)), function() {`

Comment (CrossEye) [inline src/converge.js]: Yes.   A bit myopic here.

Comment (davidchambers) [inline src/converge.js]: > as with `uncurryN`, should this use `curryN`?

Yes, it should. Good catch. :)

> The first function -- the join -- should not count when calculating the length of the returned function.

Thank you for catching this error, @CrossEye! I've updated the code and the relevant test case.

**Comment by buzzdecafe** [comment]
:coffee:

**Comment by CrossEye** [comment]
:herb:

**Comment by anton-rudeshko** [comment]
:+1:

**Pull request merged**